cypress/e2e/tests/migration/applicationinventory/assessment/miscellaneous.test.ts

```diff
@@ -157,6 +157,24 @@ describe(["@tier3"], "Tests related to application assessment and review", () =>
         // AssessmentQuestionnaire.delete(cloudNative);
     });
 
+    it("Discards application assessment through Assessment Actions page", function () {
+        // Polarion TC MTA-440
+        AssessmentQuestionnaire.enable(legacyPathfinder);
+        Application.open();
```

Comment:
This is not required . 
application.create will open application page .

Reply:
done. removed this line of code